### PR TITLE
improve: faqs: Clarify content of home/user vs install directory

### PR DESCRIPTION
**Pull request by kingthorin**
I also removed the WinXP info as it's long past end of life, and cleaned up whitespace and code fencing.

Fixes zaproxy/zaproxy-website#508

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: site/content/faq/what-is-the-default-directory-that-zap-uses.md [hunk 1]
```diff
@@ -7,35 +7,21 @@ weight: 2
 
 The default directory that ZAP uses depends on the OS.
 
-Note that you can override this using the -dir [command line](/docs/desktop/cmdline/) option.
+Note that you can override this using the `-dir` [command line](/docs/desktop/cmdline/) option.
 
-##  Windows 7 / 8
+This 'home' or 'user' directory contains ZAP files added or modified at runtime. Including (but not limited to): config files, add-ons, input files, etc. It is also the initial default save location.
```
Comment (thc202) [inline site/content/faq/what-is-the-default-directory-that-zap-uses.md:12]: `This` → `The`? (Same comment for the other FAQ entry.)

File: site/content/faq/where-is-zap-installed.md [hunk 2]
```diff
@@ -5,25 +5,16 @@ category: General Questions
...
-Underneath the 'Program Files (x86)' directory, e.g.
+This install directory contains everything that's bundled with ZAP originally.
 
-    
-    
-    C:\Program Files (x86)\OWASP\Zed Attack Proxy
-    
-
-##  Windows XP
+##  Windows 7
```
Comment (thc202) [inline site/content/faq/where-is-zap-installed.md:12]: Can this have `8 / 10` like the other?

**Comment by kingthorin** [comment]
Done & done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by kingthorin** [comment]
Added "logs" to the home bit.

File: site/content/faq/what-is-the-default-directory-that-zap-uses.md [hunk 3]
```diff
@@ -7,35 +7,21 @@ weight: 2
 
 The default directory that ZAP uses depends on the OS.
 
-Note that you can override this using the -dir [command line](/docs/desktop/cmdline/) option.
+Note that you can override this using the `-dir` [command line](/docs/desktop/cmdline/) option.
 
-##  Windows 7 / 8
+The 'home' or 'user' directory contains ZAP files added or modified at runtime. Including (but not limited to): config files, add-ons, input files, logs, etc. It is also the initial default save location.
```
Comment (garthoid) [inline site/content/faq/what-is-the-default-directory-that-zap-uses.md:12]: Please add what is saved in the initial default save location. Eg. Scripts, contexts active scan policy files sessions. etc

**Review by garthoid — CHANGES_REQUESTED** [review]
Please add what is saved in the initial default save location. Eg. Scripts, contexts active scan policy files sessions. etc

**Comment by kingthorin** [comment]
Tweaked.

**Comment by thc202** [comment]
That's not correct/accurate. The scan polices are always saved in the home (there's no initial location), the initial location for sessions is actually the user home (not ZAP's home)...

**Comment by kingthorin** [comment]
Okay, will adjust further.

**Comment by kingthorin** [comment]
Hopefully that's more accurate/clear.

**Comment by garthoid** [comment]
> That's not correct/accurate. The scan policies are always saved in the home (there's no initial location), the initial location for sessions is actually the user's home (not ZAP's home)...

I see on my Linux ~/.ZAP/session where ~/.ZAP is the default home. This is where my default sessions are placed. Am I missing something?

~ = users home
~/.ZAP = default ZAP home

**Comment by kingthorin** [comment]
ZAP does remember where you last persisted a session, so if you've chosen that at some point then it isn't the initial value anymore.

I just checked that on Kali. Starting ZAP as I normally would the persist (save) location was a folder on the desktop. If I stated zap with `-dir` to get a clean config then the persist location was `~`. (Granted that's only one Linux OS out of 100s, but I suspect the majority to behave similarly.)

File: site/content/faq/what-is-the-default-directory-that-zap-uses.md [hunk 4]
```diff
@@ -7,35 +7,23 @@ weight: 2
 
 The default directory that ZAP uses depends on the OS.
 
-Note that you can override this using the -dir [command line](/docs/desktop/cmdline/) option.
+It can be overridden using the `-dir` [command line](/docs/desktop/cmdline/) option.
 
-##  Windows 7 / 8
+The default or home directory contains ZAP files added or modified at runtime. Including (but not limited to): config files, add-ons, input files, logs, etc. It is also the default save location, for: scripts, active scan policy files, exported contexts, etc.
```
Comment (thc202) [inline site/content/faq/what-is-the-default-directory-that-zap-uses.md:12]: Saying "default" for policies it's still not correct to me, that implies that there are more locations but there's only one.
Anyway, IMHO it's unnecessary to go into much details in the FAQ, we can document the defaults or actual locations for each "file" in the corresponding user guide section.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: site/content/faq/what-is-the-default-directory-that-zap-uses.md (hunk 4 above)
Comment (kingthorin) [reply site/content/faq/what-is-the-default-directory-that-zap-uses.md:12]: Your point makes sense, I'll drop the "default"

Comment (kingthorin) [reply site/content/faq/what-is-the-default-directory-that-zap-uses.md:12]: > we can document the defaults or actual locations for each "file" in the corresponding user guide section.

Agreed, I was just trying to give a minimal explanation that people could refer to.

**Comment by thc202** [comment]
Thank you again.

File: site/content/faq/what-is-the-default-directory-that-zap-uses.md [hunk 5]
```diff
@@ -7,35 +7,23 @@ weight: 2
 
 The default directory that ZAP uses depends on the OS.
 
-Note that you can override this using the -dir [command line](/docs/desktop/cmdline/) option.
+It can be overridden using the `-dir` [command line](/docs/desktop/cmdline/) option.
 
-##  Windows 7 / 8
+The default or home directory contains ZAP files added or modified at runtime. Including (but not limited to): config files, add-ons, input files, logs, etc. It is also the save location, for: scripts, active scan policy files, exported contexts, etc.
```
Comment (psiinon) [inline site/content/faq/what-is-the-default-directory-that-zap-uses.md:12]: It is also the _default_ save location ... - users can change that

Comment (thc202) [reply site/content/faq/what-is-the-default-directory-that-zap-uses.md:12]: But not for everything mentioned.

Comment (kingthorin) [reply site/content/faq/what-is-the-default-directory-that-zap-uses.md:12]: Per comments yesterday I thought the default save location was user home (not zap home).

Comment (kingthorin) [reply site/content/faq/what-is-the-default-directory-that-zap-uses.md:12]: If someone else wants to suggest different text I'm happy to update the PR.

**Review by psiinon — APPROVED** [review]

**Pull request merged**